python/tvm/topi/hexagon/utils.py

```diff
@@ -151,3 +152,81 @@ def get_layout_transform_fn(layout):
     if layout == "nhwc-8h8w32c-2d":
         return nhwc_8h8w32c_2d
     raise RuntimeError(f"Unexpected layout '{layout}'")
+
+
+def get_fixed_point_value(flp, dtype="int16"):
```

Comment:
I copied the function from this PR: https://github.com/apache/tvm/pull/12340/files#diff-1697dbb756a60c67fcd89181b9fc9193d44383894adf22ff17a2b7f59255b766R165

If the PR gets merged before mine, I will remove this from my PR